waspc/waspc.cabal

```diff
@@ -323,6 +323,7 @@ library cli-lib
     , optparse-applicative ^>=0.17.0.0
```

Comment:
Hey, trying out this locally after `cabal install` @sodic and I noticed that we need to update our `data-files` as well.

This list worked for us, but you might want to double check if everything is working:
```
  packages/deploy/dist/**/*.js
  packages/deploy/node_modules/**/*.js
  packages/deploy/node_modules/**/*.mjs
  packages/deploy/node_modules/**/*.cjs
  packages/deploy/node_modules/**/*.json
  packages/deploy/package.json
  ```
  
  Try it out like this:
  ```bash
cabal install --overwrite-policy=always
gwasp=/Users/ilakovac/.cabal/bin/wasp-cli
```
and then
```
gwasp deploy
```
in project dir.

Reply:
Thanks for trying it out locally. We don't need to update the cabal file since we sneakily add the TS project to the data dir in CI. I tried to call out how to do so in the PR notes above, but it was definitely easy to miss. Here is the relevant part:

> 🆘 REQUEST FOR HELP: If reviewers wouldn't mind trying it out themselves, that would be super helpful 😅 Especially interested may be @vincanger. Here is how you can test locally:
> ```
> wasp/waspc $ ./tools/install_deploy_package_to_data_dir.sh
> wasp/waspc $ cd examples/todoApp
> wasp/waspc/examples/todoApp $ cabal run wasp-cli deploy fly <cmds>
> ```

If you have any questions about that part though, just let me know 👍🏻

Reply:
Sorry about the confusion 👍 If you are copying the packages folder in CI, it should work. One thing we noticed when we copied only the files listed above, the package folder went from 108MB down to 78MB.

108MB would then need to be downloaded each time user installs Wasp 😢 
Maybe we can optimize that later (going for the npm package `@wasp/deploy-fly` after all) in some other PR.

Reply:
It looks like above you all included `node_modules`. I remove those in CI before copying the compiled TS project into the data dir, and then we `NPM` install them on first run (since including `node_modules` would be large and possibly cause issues for the user if there are any dynamic linking differences between our CI machine and their host OS).

The actual size of what we include in the installer artifact should only be:
```
$ du -sh packages/
440K    packages/
```

Ref: https://github.com/wasp-lang/wasp/blob/shayne-deploy-packaged/waspc/tools/install_deploy_package_to_data_dir.sh#L12

Reply:
Again, sorry for not understanding you went through the failure scenarios already 👍 This is great!

Reply:
No sweat at all, I appreciate your checking on it!

Reply:
Plus 1 on this being awesome!